TestPlaybooks/playbook-GmailTest.yml

```diff
@@ -166,7 +166,7 @@ tasks:
       - "21"
     scriptarguments:
       email:
-        simple: JohnS@demistodev.com
+        simple: JohnS1@demistodev.com
```

Comment:
better to manually delete the user from the instance (deleted it myself)

Reply:
No problem, will revert the change.
Just for future knowledge - why is that better? Meaning, what are the considerations?